### ISSUE TITLE
save_csv doesn't work

### ISSUE THREAD

**Issue by mmitchef** [ProGamerGov/dream-creator#27]
I am trying to save the accuracy for each class but it doesn't work. Do you have any idea how I can fix this. 

```
-save_csv: Enabling this flag will save loss and accuracy data to txt files.
-csv_dir: Where to save csv files. Default is set to current working directory.
```

inside of utils/train_model.py you have these lines of code 
```
def train_model(model, dataloaders, criterion, optimizer=None, lrscheduler=None, num_epochs=120, start_epoch=1, save_epoch=1, \
                output_name='out.pth', device='cuda:0', has_branches=False, fc_only=False, num_classes=0, individual_acc = False, \
                should_save_csv=False, csv_path='', save_info=None):
```

**Comment by mmitchef** [comment]
To get this work, 

1. inside of `'dream-creator/train_googlenet.py'` change this line `should_save_csv=params.save_csv, csv_path=params.csv_dir, save_info=save_info)` to `save_csv=params.save_csv, csv_dir=params.csv_dir, save_info=save_info)
2.  inside of dream-creator/utils/training_utils.py  change `# Save list as CSV
def save_csv_data(csv_path, filename, new_line):
    with open(os.path.join(csv_path, filename), "a+", newline='') as f:
        wr = csv_writer(f, quoting=QUOTE_ALL)
        wr.writerow(new_line)` to /```
# Save list as CSV
def save_csv_data(csv_dir, filename, new_line):
    with open(os.path.join(csv_dir, filename), "a+", newline='') as f:
        wr = csv_writer(f, quoting=QUOTE_ALL)
        wr.writerow(new_line)
```
